reference/6/Microsoft.PowerShell.Core/Unregister-PSSessionConfiguration.md

```diff
@@ -216,9 +226,8 @@ This cmdlet does not return any objects.
 
 ## NOTES
 
-* To run this cmdlet on Windows Vista, Windows Server 2008, and later versions of the Windows operating system, you must start PowerShell by using the Run as administrator option.
-
-*
+To run this cmdlet on Windows Vista, Windows Server 2008, and later versions of the Windows
```

Comment:
Vista has exited support... should we update this to something like 2008 R2 that's still in support (even though it will end in Jan, 2020) or something newer that is in support?

Reply:
fixed